packages/web/src/client/components/Blocks/BlocksCarousel/index.tsx

```diff
@@ -177,7 +177,9 @@ export const Carousel = (props: Sanity.BlockMedia) => {
         <Caption>
           {items[activeIndex].caption ? (
             <CaptionText>{items[activeIndex].caption}</CaptionText>
-          ) : null}
+          ) : (
```

Comment:
this can probably be inlined no? we're still rendering the element regardless.

Reply:
@joshuaellis 

`inlined` meaning where? I thought the `CaptionText` renders only when we have a caption. If not we have no first item in the flex and the 3 dots are taking its place and moving to the left.

Reply:
I mean couldn't we remove the ternary and just render the component always, if there's no caption, there's no caption no text will render.

Reply:
oh yes, sorry, got stack on the `inline` part